contracts/tools/Swaps.sol

```diff
@@ -94,9 +93,52 @@ contract DVP is Ownable, ERC1820Client, IERC1400TokensRecipient, ERC1820Implemen
 
   enum State {Undefined, Pending, Executed, Forced, Cancelled}
 
-  enum TradeType {Escrow, Swap}
+  enum TradeType {Allowance, Hold, Escrow}
 
   enum Holder {Holder1, Holder2}
+
+  string internal constant DVP_HOLDABLE_LOCKABLE = "DVPHoldableLockable";
```

Comment:
Not sure "DVP_HOLDABLE_LOCKABLE" and  "ERC1400_TOKENS_VALIDATOR" are used

Reply:
`ERC1400_TOKENS_VALIDATOR` is still being used in `tokensReceived` function. The `DVP_HOLDABLE_LOCKABLE` has been removed